gin/userDashHandlers.go

```diff
@@ -30,6 +30,47 @@ func handleUserDash(c *gin.Context) {
 	renderHTML(c, 200, "userdash-top.html", data)
 }
 
+func handleOrderDetail(c *gin.Context){
+	data := make(map[string]interface{})
+	data["Title"] = "Order Details"
+	var order gorm.Order
+	stuff := c.Param("stuff")
+	err := order.PopulateByUUID(stuff)
+	if err != nil {
+		logger.Error("order uuid in params is not valid:", err)
+		logger.Info("boucing back to /dash")
+		c.Redirect(http.StatusFound, "/dash")
+	}
+
+	if order.StatusCode == 60{
```

Comment:
@dpham273 don't use 60 here, instead use the constant that has been defined in gorm. In this case, `OrderStatusDelivered`